src/PockyBot.NET/PockyBotFactory.cs

```diff
@@ -28,14 +28,14 @@ public static IPockyBot CreatePockyBot(PockyBotSettings settings, IChatHelper ch
             var pegRequestValidator = new PegRequestValidator(wrappedSettings, configRepository);
             var pegHelper = new PegHelper(configRepository);
             var pegGiver = new PegGiver(pegRepository, chatHelper);
-            var directResultsMessageSender = new DirectResultsMessageSender(chatHelper.Messages);
+            var directResultsMessageSender = new DirectResultsMessageSender(chatHelper.Messages, loggerFactory.CreateLogger<DirectResultsMessageSender>());
```

Comment:
Codecov is complaining about this line. Out of curiosity I wonder if Codecov can be amended to ignore this file.